src/assignments/ravindra/Assignment1.java

```diff
@@ -3,7 +3,25 @@
 public class Assignment1 {
 
 //	Write a reusable method print Even and Odd numbers from the given range
-	public static void main(String[] args) {
-		
+	
+	
+    public static void reusableMethodPrintEvenAndOddNumbersFromTheGivenRange(int start, int end) {
```

Comment:
Rename the mthod to "printEvenAndOddNumbersForGivenRange()"

Reply:
Addressed the comments and changes done accordingly.